src/lib/components/Toast/ToastItem.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import Toast from './Toast';
+
+const ToastItem = ({ children, remove, options }) => {
```

Comment:
O `ToastItem` na verdade encapsula um `Toast` dentro dele e da ao dev o poder de gerenciar o estado do seu toast. Seria legal pensar em uma nomenclatura que fique mais clara para quem for dar manutenção nesse cara. `ToastItem` da a impressão que é um subcomponente de `Toast`, que vai dentro dele, quando na verdade ele funciona mais como um wrapper. Inclusive, `ToastWrapper` não ficaria legal?